### PR TITLE
Refactor tests so none need to be ignored

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: test/integration/local/message-service.test.js [hunk 1]
```diff
@@ -11,41 +15,50 @@ const generateSampleClaim = () => ({
...
-    const message = generateSampleClaim()
-    const scheduleSender = messageService.scheduleSender
-    const spy = jest.spyOn(scheduleSender, 'sendMessage')
+  test('Processed message ends up on scheduleQueue', async () => {
+    let done
+    const promise = new Promise((resolve) => {
+      done = resolve
+    })
+    const action = (result) => {
+      done(result.hello === message.hello)
```
Comment (markysoft) [inline test/integration/local/message-service.test.js:45]: there is no `hello` property on the result object, it should probably be checking on `content`

File: test/integration/local/message-service.test.js [hunk 2]
```diff
@@ -11,41 +15,50 @@ const generateSampleClaim = () => ({
...
-    const message = generateSampleClaim()
-    const calculationSender = messageService.calculationSender
-    const spy = jest.spyOn(calculationSender, 'sendMessage')
+  test('Processed message ends up on calculationQueue', async () => {
+    let done
+    const promise = new Promise((resolve) => {
+      done = resolve
+    })
+    const action = (result) => {
+      done(result.hello === message.hello)
```
Comment (markysoft) [inline test/integration/local/message-service.test.js:58]: similar to the above, `hello` will always be `undefined`

**Review by markysoft — CHANGES_REQUESTED** [review]
There was an error in the original test implementation that can be addressed.

File: test/integration/local/message-service.test.js (hunk 2 above)
Comment (st3v3nhunt) [reply test/integration/local/message-service.test.js:58]: You can't beat a test for `undefined === undefined`! 😞

**Review by markysoft — APPROVED** [review]

**Pull request merged**